### ISSUE TITLE
Move to Behat 3

### ISSUE THREAD

**Issue by tyx** [PolishSymfonyCommunity/Symfony2MockerExtension#6]
Hi,

Any plan to move this extension to behat 3 ?

I can help on this topic.

**Comment by jdecool** [comment]
Any news about this issue ?

**Comment by tyx** [comment]
Just for information, we made this : https://github.com/rezzza/mock-extension

We don't support Mockery for now, but atoum is really fine ;)

**Comment by jdecool** [comment]
Thanks @tyx I'm going to look at that.

No problem for atoum, it's the tool I use :)

**Comment by yoanm** [comment]
Up !
Any plan ? At least add a contraint on your composer.json (`2.*` instead of `>=2.4`)

**Comment by jakzal** [comment]
The master branch now contains a version that's compatible with Behat 3. Can you please try it out?

```
composer require polishsymfonycommunity/symfony2-mocker-extension:dev-master
```

**Comment by yoanm** [comment]
Thanks for your reactivity !

Unfortunately, it was for a company work. And so, to not loosing time, we have already switch to our custom implementation for behat tests, and have no time to test if it's works or not :/.

Anyway, I'll keep your project in minds.
Thanks again.

**Comment by lrocheWB** [comment]
Hi,
```
polishsymfonycommunity/symfony2-mocker-extension dev-master requires mockery/mockery ^0.7.0 -> no matching package found
```

**Comment by jakzal** [comment]
@lrocheWB is this the full message?

**Comment by lrocheWB** [comment]
Hi @jakzal,
Sorry for the delay, yes it's full message.
Do you reproduce ?

**Comment by jakzal** [comment]
@lrocheWB this should have been fixed by the latest version in master

**Comment by jakzal** [comment]
Just released [v2.0.0](https://github.com/PolishSymfonyCommunity/Symfony2MockerExtension/releases/tag/v2.0.0). 

In the meantime let me introduce you to https://github.com/docteurklein/TestDoubleBundle, which follows a less hackish approach.
